tracer/src/Datadog.AzureFunctions/Datadog.AzureFunctions.csproj

```diff
@@ -0,0 +1,90 @@
+    <NugetAudit>true</NugetAudit>
+    <CheckEolTargetFramework>true</CheckEolTargetFramework>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\Datadog.Trace.Manual\Datadog.Trace.Manual.csproj" />
+    <ProjectReference Include="$(MSBuildThisFileDirectory)..\Datadog.Trace.Annotations\Datadog.Trace.Annotations.csproj" />
+
+    <PackageReference Include="Datadog.Serverless.Compat" Version="0.1.0" PrivateAssets="none" />
+
+    <!-- managed assemblies -->
+    <None Include="$(SourcePath)/net6.0/Datadog.Trace.dll" Pack="true" Visible="false" PackagePath="contentFiles/any/any/datadog/net6.0/">
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </None>
+    <None Include="$(SourcePath)/net461/Datadog.Trace.dll" Pack="true" Visible="false" PackagePath="contentFiles/any/net461/datadog/net461/">
+      <Link>home/net461/Datadog.Trace.dll</Link>
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </None>
+
+    <!-- native libraries -->
+    <None Include="$(SourcePath)/win-x64/*" Pack="true" Visible="false" PackagePath="contentFiles/any/any/datadog/win-x64/;contentFiles/any/net461/datadog/win-x64/">
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </None>
+    <None Include="$(SourcePath)/linux-x64/*" Pack="true" Visible="false" PackagePath="contentFiles/any/any/datadog/linux-x64/">
+      <PackageCopyToOutput>true</PackageCopyToOutput>
+    </None>
+
+    <!-- extras -->
+    <None Include="README.md" Pack="true" Visible="true" PackagePath="/">
+      <PackageCopyToOutput>false</PackageCopyToOutput>
+    </None>
+
+    <!-- win-x64 exclusions -->
+    <None Remove="$(SourcePath)/win-x64/Datadog.Profiler.Native.dll" />       <!-- 1.3 MB -->
+    <None Remove="$(SourcePath)/win-x64/datadog_profiling_ffi.dll" />         <!-- 5.0 MB -->
+    <None Remove="$(SourcePath)/win-x64/dd-dotnet.exe" />                     <!-- 3.3 MB -->
+    <None Remove="$(SourcePath)/win-x64/ddwaf.dll" />                         <!-- 3.6 MB -->
+
+    <!-- linux-x64 exclusions -->
+    <None Remove="$(SourcePath)/linux-x64/Datadog.Linux.ApiWrapper.x64.so" /> <!-- 0.02 MB -->
+    <None Remove="$(SourcePath)/linux-x64/Datadog.Profiler.Native.so" />      <!-- 2.7  MB -->
+    <None Remove="$(SourcePath)/linux-x64/dd-dotnet" />                       <!-- 3.8  MB -->
+    <None Remove="$(SourcePath)/linux-x64/libdatadog_profiling.so" />         <!-- 8.4  MB -->
+    <None Remove="$(SourcePath)/linux-x64/libddwaf.so" />                     <!-- 2.5  MB -->
+
+    <!-- include trimming files -->
+    <None Include="$(MSBuildThisFileDirectory)../Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.props" Pack="true" PackagePath="buildTransitive/net6.0/$(PackageId).props" />
+    <None Include="$(MSBuildThisFileDirectory)../Datadog.Trace.Trimming/build/Datadog.Trace.Trimming.xml" Pack="true" PackagePath="buildTransitive/net6.0/" />
```

Comment:
These are included in Datadog.Trace so are not required I think

Reply:
In the `Datadog.Trace` package, they live in `build`, not in `buildTransitive`, so they are not transitive. (We should fix that someday.)

Reply:
Thanks. Since that change is released now 3.20.0, I removed the trimming files here.